rasa/utils/train_utils.py

```diff
@@ -95,22 +95,24 @@ def align_tokens(
     current_token_offset = token_start
 
     for index, string in enumerate(tokens_in):
+        string_len = len(string) if string != "[UNK]" else 1
```

Comment:
Where does the `"[UNK]"` token come from? Would it be always `"[UNK]"` or can it also be something else?

Reply:
Also why should the length always be 1? Can it also happen that the orginial word has a length longer than 1? Would that not break?

Reply:
Hi Tanja, "[UNK]" is the default OOV token of BERT. It's my fault to think that all tokenizers use the same OOV token, after more investigation, I found that different models may have different OOV token settings. Luckily, all those tokenizers used in Rasa inherit from PreTrainedTokenizer, which we can get OOV token by access the attribute `unk_token` of the instance. I will change the interface of align_tokens, make sure it always working on the correct OOV token.

The length of the OOV token is not guaranteed always be 1, some documents (https://mccormickml.com/2019/05/14/BERT-word-embeddings-tutorial/#22-tokenization; https://github.com/huggingface/tokenizers) gives the information that it seems that only emoji character can cause OOV in  English and most many other languages. Tokenizer for Chinese may treat the character as the token but Chinese has lots of characters, so OOV still occurs sometimes, and this patch will be very helpful to such a situation.